modules/oc-mirror-oci-format.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="oc-mirror-oci-format_{context}"]
+= Mirroring images in OCI format
+
+You can use the oc-mirror plug-in to mirror Operators in the Open Container Initiative (OCI) image format, instead of Docker v2 format. You can copy Operator images to a file-based catalog on disk in OCI format. Then you can then copy local OCI images to your target mirror registry.
```

Comment:
@sherine-k @lmzuccarelli This intro/details section is a little light. Please let me know if there is other information that we should include here.